settings.py

```diff
@@ -0,0 +1,772 @@
+    savefile: Optional[str] = None
+    disable_save: bool = False
+    loglevel: str = "info"
+    server_password: Optional[ServerPassword] = None
+    disable_item_cheat: Union[DisableItemCheat, bool] = False
+    location_check_points: LocationCheckPoints = LocationCheckPoints(1)
+    hint_cost: HintCost = HintCost(10)
+    release_mode: ReleaseMode = ReleaseMode("goal")
+    collect_mode: CollectMode = CollectMode("goal")
+    remaining_mode: RemainingMode = RemainingMode("goal")
+    auto_shutdown: AutoShutdown = AutoShutdown(0)
+    compatibility: Compatibility = Compatibility(2)
+    log_network: LogNetwork = LogNetwork(0)
+
+
+class GeneratorOptions(Group):
+    """Options for Generation"""
+
+    class EnemizerPath(LocalFilePath):
+        """Location of your Enemizer CLI, available here: https://github.com/Ijwu/Enemizer/releases"""
+        is_exe = True
+
+    class PlayerFilesPath(OptionalUserFolderPath):
+        """Folder from which the player yaml files are pulled from"""
+        # created on demand, so marked as optional
+
+    class Players(int):
+        """amount of players, 0 to infer from player files"""
+
+    class WeightsFilePath(str):
+        """
+        general weights file, within the stated player_files_path location
+        gets used if players is higher than the amount of per-player files found to fill remaining slots
+        """
+        # this is special because the path is relative to player_files_path
+
+    class MetaFilePath(str):
+        """Meta file name, within the stated player_files_path location"""
+        # this is special because the path is relative to player_files_path
+
+    class Spoiler(IntEnum):
+        """
+        Create a spoiler file
+        0 -> None
+        1 -> Spoiler without playthrough or paths to playthrough required items
+        2 -> Spoiler with playthrough (viable solution to goals)
+        3 -> Spoiler with playthrough and traversal paths towards items
+        """
+        NONE = 0
+        BASIC = 1
+        PLAYTHROUGH = 2
+        FULL = 3
+
+    class GlitchTriforceRoom(IntEnum):
+        """
+        Glitch to Triforce room from Ganon
+        When disabled, you have to have a weapon that can hurt ganon (master sword or swordless/easy item functionality
+        + hammer) and have completed the goal required for killing ganon to be able to access the triforce room.
+        1 -> Enabled.
+        0 -> Disabled (except in no-logic)
+        """
+        OFF = 0
+        ON = 1
+
+    class PlandoOptions(str):
+        """
+        List of options that can be plando'd. Can be combined, for example "bosses, items"
+        Available options: bosses, items, texts, connections
+        """
+
+    class Race(IntEnum):
+        """Create encrypted race roms and flag games as race mode"""
+        OFF = 0
+        ON = 1
+
+    enemizer_path: EnemizerPath = EnemizerPath("EnemizerCLI/EnemizerCLI.Core")  # + ".exe" is implied on Windows
```

Comment:
Just found one small issue, actually.

This does not resolve on Windows. Would have to check if windows use "\\" instead of "/" or if Linux can handle it fine, just use "EnemizerCLI\\EnemizerCLI.Core".

Reply:
The problem was not the '/', but that missing automatic ".exe". It converted from `settings.Path` to `str` too early. Pushed a fix.